app/views/config_reports/welcome.html.erb

```diff
@@ -1,16 +1,29 @@
+<% content_for(:title, _("Reports")) %>
+
 <div class="blank-slate-pf">
   <div class="blank-slate-pf-icon">
     <%= icon_text("book", "", :kind => "fa") %>
   </div>
   <h1><%= _('Reports') %></h1>
-  <p>
-  <%= _("You don't seem to have any reports.") %></br>
-  <%= _('If you wish to configure Puppet to forward it reports to Foreman, please follow') %>
-  <%= link_to _("setting up reporting"), documentation_url("3.5.4PuppetReports"), :rel => "external" %>
-  <%= _("and") %>
-  <%= link_to _("e-mail reporting"), "http://theforeman.org/projects/foreman/wiki/Mail_Notifications", :rel => "external" %>.
-  </p>
-  <div class="blank-slate-pf-main-action">
-    <%= link_to _('Documentation'), documentation_url("3.5.4PuppetReports"), :rel => 'external', :class => 'btn btn-primary btn-lg' %>
-  </div>
+  <% if Setting[:enable_reports_storage] %>
+    <p>
+      <%= _("You don't seem to have any reports.") %></br>
+      <%= _('If you wish to configure Puppet to forward it reports to Foreman, please follow') %>
+      <%= link_to _("setting up reporting"), documentation_url("3.5.4PuppetReports"), :rel => "external" %>
+      <%= _("and") %>
+      <%= link_to _("e-mail reporting"), "http://theforeman.org/projects/foreman/wiki/Mail_Notifications", :rel => "external" %>.
```

Comment:
Not sure if there's a helper to link to the wiki, but it should at least link to the correct URL

```suggestion
      <%= link_to _("e-mail reporting"), "https://projects.theforeman.org/projects/foreman/wiki/Mail_Notifications", :rel => "external" %>.
```